profile/README.md

```diff
@@ -0,0 +1,27 @@
+<p align="center">
+  <img src="https://avatars.githubusercontent.com/u/871554?s=200&v=4" alt="Betterment's profile image"/>
```

Comment:
Get better image to use here

Reply:
Can maybe done as a follow-up. Whatever it ends up being should be dark-mode friendly cuz rn it isn't!